cos-connector-camel/runtime/src/main/java/org/bf2/cos/connector/camel/ConnectorRecorder.java

```diff
@@ -22,7 +21,7 @@ public ConnectorContextCustomizer(ConnectorConfig config) {
 
         @Override
         public void configure(CamelContext camelContext) {
-            camelContext.getGlobalOptions().put(JacksonConstants.ENABLE_TYPE_CONVERTER, "true");
+            //camelContext.getGlobalOptions().put(JacksonConstants.ENABLE_TYPE_CONVERTER, "true");
```

Comment:
Is that correct?

Reply:
Yes, that is the actual reason of the issue.
I need to remove that line